### ISSUE TITLE
The application Atom has closed unexpectedly

### ISSUE THREAD

**Issue by joefitzgerald** [atom/atom#11172]
### Prerequisites

* ~~[ ] Can you reproduce the problem in [safe mode](https://atom.io/docs/latest/hacking-atom-debugging#check-if-the-problem-shows-up-in-safe-mode)?~~ Cannot reproduce.
* [x] Are you running the [latest version of Atom](https://atom.io/docs/latest/hacking-atom-debugging#update-to-the-latest-version)?
* [x] Did you check the [debugging guide](https://atom.io/docs/latest/hacking-atom-debugging)?
* [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

### Description

Atom crashed on Linux when installed on the https://atlas.hashicorp.com/box-cutter/boxes/ubuntu1404-desktop vagrant box.

### Steps to Reproduce

`Vagrantfile`
```ruby
Vagrant.configure(2) do |config|
  config.vm.box = "box-cutter/ubuntu1404-desktop"

  config.vm.provider "virtualbox" do |v|
    v.gui = true
  end
end

```

1. Go to atom.io, and install atom
2. Launch atom
3. Close atom via the `X` icon in the top left

**Note: ** I cannot reproduce this.

**Expected behavior:** No crashing.

**Actual behavior:** Crashed:

![screen shot 2016-03-15 at 8 24 20 am](https://cloud.githubusercontent.com/assets/744740/13781203/2855b2b6-ea88-11e5-9556-5d3ee95560d0.png)

### Versions

```
$ atom --version 
1.5.4

$ apm --version
apm  1.6.0
npm  2.13.3
node 0.10.40
python 2.7.6
git 1.9.1
```

**Comment by joefitzgerald** [comment]
Just happened again on stable (1.6).

![screen shot 2016-03-21 at 9 03 33 am](https://cloud.githubusercontent.com/assets/744740/13922866/d212d490-ef43-11e5-9a89-73ed0007b7db.png)

**Comment by Ben3eeE** [comment]
Just had this crash or similar happen on about as fresh install of ubuntu and atom as you can get in a VM.
No packages installed completely stock atom installed from deb downloaded from atom.io.
VM installed less than an hour ago only things I installed was atom and virtualbox-guest-dkms. After which I rebooted.
First launch of atom and less than a minute later when I started closing the welcome tabs it crashed. What is strange is that I am sure that I did not close atom just the tabs but it was lagging so maybe I am wrong.

Can not reproduce.

![atom closed unexpectedly](https://cloud.githubusercontent.com/assets/1058982/14435501/76c36d3a-0018-11e6-8c8e-c9c1ea3f8682.PNG)

Ubuntu 15.10 amd64 Oracle VM VirtualBox. Windows 7 host os.
atom --version 1.6.2
apm --version
apm 1.6.0
npm 2.13.3
node 0.10.40
python 2.7.10
git 2.5.0

**Comment by Ben3eeE** [comment]
Had this happen a few times now but all my times were when starting atom, not when closing it. The error looks the same but caused by a different thing.
Every time I have seen this crash I have also had the Ubuntu software updater start and remind me that I have updates to install.

**Comment by Scimonster** [comment]
I occasionally get this when closing atom.

```
$ atom --version
Atom    : 1.10.2
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0
$ apm --version
apm  1.12.5
npm  3.10.5
node 4.4.5
python 2.7.12
git 2.7.4
```
Ubuntu 16.04.

**Comment by joefitzgerald** [comment]
@Scimonster It would be beneficial if you could find a way to reproduce the issue. Additionally, if you run https://atom.io/beta, do you still experience this behavior?

**Comment by SylannBin** [comment]
It has been happening to me on ubuntu 16.04 for several months already. Ii is very likely that it happens everytime I close the application (whatever the way), but I won't swear it.

I always use latest stable build of atom.
I just reinstalled a ubuntu and downloaded atom, node, npm, ... a couple of hours ago.
For the record I decided to test right at the beginning if it would still happen.
I did not even installed any pluggin to atom yet, but it still crashed after I closed it regularly, twice in a row.

`$ atom --version
Atom    : 1.10.2
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0`

`$ apm --version
apm  1.12.5
npm  3.10.5
node 4.4.5
python 2.7.12
git 2.7.4`

capture of the top part of the crash window.
![atom bug](https://cloud.githubusercontent.com/assets/17208592/18894089/250c74c2-8511-11e6-825b-cdb851002a40.png)
